tests/playwright/src/specs/kubernetes.spec.ts

```diff
@@ -130,12 +133,94 @@ test.describe('Kubernetes resources End-to-End test', () => {
         .toEqual(KubernetesResourceState.Running);
     });
     test('Delete the PVC resource', async ({ page, navigationBar }) => {
-      await deletePod(page, POD_NAME);
+      await deletePod(page, PVC_POD_NAME);
       const kubernetesBar = await navigationBar.openKubernetes();
       const pvcsPage = await kubernetesBar.openTabPage(KubernetesResources.PVCs);
       await pvcsPage.deleteKubernetesResource(PVC_NAME);
       await handleConfirmationDialog(page);
       await playExpect(pvcsPage.getResourceRowByName(PVC_NAME)).not.toBeVisible();
     });
   });
+  test.describe.serial('ConfigMaps and Secrets lyfecycle test', () => {
+    test('Create ConfigMap and Secret resources', async ({ navigationBar }) => {
+      const podsPage = await navigationBar.openPods();
+      await playExpect(podsPage.heading).toBeVisible();
+      const playYamlPage = await podsPage.openPlayKubeYaml();
+      await playExpect(playYamlPage.heading).toBeVisible();
+      const configmapYamlFilePath = path.resolve(
+        __dirname,
+        '..',
+        '..',
+        'resources',
+        'kubernetes',
+        `${CONFIGMAP_NAME}.yaml`,
+      );
+      await playYamlPage.playYaml(configmapYamlFilePath, {
+        runtime: PlayYamlRuntime.Kubernetes,
+        kubernetesContext: KUBERNETES_CONTEXT,
+        kubernetesNamespace: KUBERNETES_NAMESPACE,
+      });
+
+      await playExpect(podsPage.heading).toBeVisible();
+      await podsPage.openPlayKubeYaml();
+      const secretYamlFilePath = path.resolve(__dirname, '..', '..', 'resources', 'kubernetes', `${SECRET_NAME}.yaml`);
+      await playYamlPage.playYaml(secretYamlFilePath, {
+        runtime: PlayYamlRuntime.Kubernetes,
+        kubernetesContext: KUBERNETES_CONTEXT,
+        kubernetesNamespace: KUBERNETES_NAMESPACE,
+      });
+
+      const kubernetesBar = await navigationBar.openKubernetes();
+      const configmapSecretsPage = await kubernetesBar.openTabPage(KubernetesResources.ConfigMapsSecrets);
+      await playExpect(configmapSecretsPage.heading).toBeVisible();
+      await playExpect(configmapSecretsPage.getResourceRowByName(CONFIGMAP_NAME)).toBeVisible();
+      const configmapDetails = await configmapSecretsPage.openResourceDetails(
```

Comment:
consider dividing the each test into two - resource create and resource details check or something. I will let that to you.